tests/cli/commands/test_dag_command.py

```diff
@@ -497,12 +497,13 @@ def test_cli_list_dag_runs(self):
             [
                 'dags',
                 'list-runs',
+                '--dag-id',
+                'example_bash_operator',
                 '--no-backfill',
                 '--start-date',
                 DEFAULT_DATE.isoformat(),
                 '--end-date',
                 timezone.make_aware(datetime.max).isoformat(),
-                'example_bash_operator',
             ]
```

Comment:
Hmmmm, this feels like a quasi-breaking change. What worked on Airflow 2.3 won't work anymore.

Reply:
This is undoing a breaking change that is only in main (see the [revert commit](https://github.com/apache/airflow/pull/26357/commits/bef2752d96b17b3d677e92f0516bfe3136d5e6dd)).

2.3: https://github.com/apache/airflow/blob/88b274c95b212b541ba19918880ae425856212be/tests/cli/commands/test_dag_command.py#L447-L448

Reply:
But yes, this is exactly why this PR exists 👍